libcontainer/configs/validate/validator.go

```diff
@@ -153,6 +157,19 @@ func (v *ConfigValidator) sysctl(config *configs.Config) error {
 	return nil
 }
 
+func (v *ConfigValidator) intelrdt(config *configs.Config) error {
+	if config.IntelRdt != nil {
+		if !intelrdt.IsIntelRdtEnabled() {
+			return fmt.Errorf("intelRdt is specified in config, but Intel RDT is not supported")
+		}
+		if config.IntelRdt.L3CacheSchema == "" {
```

Comment:
@yummypeng 
> So, the following configuration is illegal?

In my opinion, if we specify "intelRdt" in config, we'd better specify a non-empty string for sub-item "l3CacheSchema".